src/scenes/list/itemList/ItemList.js

```diff
@@ -22,7 +23,7 @@ this.renderButton = (done, button) => {
   );
 };
 
-const ItemList = (props) => {
+const ItemList = observer((props) => {
```

Comment:
Why is this component an observer?

Reply:
The FlatList render that component, so if a make changes in one item like toggle them it needs to be observer because if it isn't it doesn't re render.

Reply:
About the version of mobs, its on version 5 actually and I use version 3 because the latest has some issues with react native